integ-test/src/test/java/org/opensearch/sql/calcite/remote/CalciteExpandCommandIT.java

```diff
@@ -0,0 +1,68 @@
+/*
+ *
+ *  * Copyright OpenSearch Contributors
+ *  * SPDX-License-Identifier: Apache-2.0
+ *
+ */
+
+package org.opensearch.sql.calcite.remote;
+
+import static org.opensearch.sql.legacy.TestsConstants.TEST_INDEX_ARRAY;
+import static org.opensearch.sql.legacy.TestsConstants.TEST_INDEX_NESTED_SIMPLE;
+import static org.opensearch.sql.util.MatcherUtils.schema;
+import static org.opensearch.sql.util.MatcherUtils.verifyNumOfRows;
+import static org.opensearch.sql.util.MatcherUtils.verifySchema;
+
+import org.json.JSONObject;
+import org.junit.Ignore;
+import org.junit.jupiter.api.Test;
+import org.opensearch.sql.ppl.PPLIntegTestCase;
+
+public class CalciteExpandCommandIT extends PPLIntegTestCase {
+  @Override
+  public void init() throws Exception {
+    super.init();
+    loadIndex(Index.NESTED_SIMPLE);
+    loadIndex(Index.ARRAY);
+    enableCalcite();
+    disallowCalciteFallback();
+  }
+
+  @Test
+  public void testExpandOnNested() throws Exception {
+    JSONObject response =
+        executeQuery(String.format("source=%s | expand address", TEST_INDEX_NESTED_SIMPLE));
+    verifySchema(
+        response,
+        schema("name", "string"),
+        schema("age", "bigint"),
+        schema("id", "bigint"),
+        schema("address", "struct"));
+    verifyNumOfRows(response, 11);
+  }
+
+  // TODO: confirm if expand on array (instead of nested) will be supported.
```

Comment:
We can wait user's feedback in future. I don't either think we can support it now.